### PR TITLE
fix/serialisation

### PR DESCRIPTION
**Pull request by Dmdv**
## Tests 
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

**Review by peterlimg — CHANGES_REQUESTED** [review]
Code changes looks good to me.

For the `0chain_loc_deployment.sh`, I suggest to keep it locally and send to the people who need to deploy the full local environment, or keep it in a separate project with document.

File: docker.local/config/sc.yaml [hunk 1]
```diff
@@ -247,11 +247,13 @@ smart_contracts:
       delete: 100
       vestingsc-update-settings: 100
   zcnsc:
+    owner_id: 1746b06bb09f55ee01b33b5e2e055d6cc7a900cb57c0a3a5eaabb8a0e7745802
     min_mint_amount: 1
-    percent_authorizers: 0.7
-    min_authorizers: 1
     min_burn_amount: 1
     min_stake_amount: 0
+    min_lock_amount: 0
```
Comment (peterlimg) [inline docker.local/config/sc.yaml]: Use `min_lock` and `min_stake` to keep the consistent naming as other SC settings.

**Review by peterlimg — DISMISSED** [review]

File: docker.local/config/sc.yaml (hunk 1 above)
Comment (Dmdv) [reply docker.local/config/sc.yaml]: done

**Comment by peterlimg** [comment]
@Dmdv please remove the `0chain_loc_deployment.sh` script from commit

**Review by peterlimg — APPROVED** [review]
lgtm

**Pull request merged**